Cannot get model of name accessToken from app.models

loopback 2.14.0
loopback-testing 1.1.0

my test file:

        var lt = require('loopback-testing');
        var should = require('should');
        var app = require('../server/server.js');
        
        describe('/jobs', function () {
          lt.beforeEach.withApp(app);
          lt.it.shouldBeAllowedWhenCalledAnonymously('GET', '/api/jobs/list');
        });

Getting error "AssertionError: cannot get model of name accessToken from app.models", looks like the internally defined model is named "AccessToken" (capitalized) instead of "accessToken".

--- a/lib/helpers.js
+++ b/lib/helpers.js
@@ -222,11 +222,11 @@ _beforeEach.givenLoggedInUserWithRole = function(credentials, role, optionalHand
 }
 
 _beforeEach.givenAnUnauthenticatedToken = function(attrs, optionalHandler) {
-  _beforeEach.givenModel('accessToken', attrs, optionalHandler);
+  _beforeEach.givenModel('AccessToken', attrs, optionalHandler);
 }
 
 _beforeEach.givenAnAnonymousToken = function(attrs, optionalHandler) {
-  _beforeEach.givenModel('accessToken', {id: '$anonymous'}, optionalHandler);
+  _beforeEach.givenModel('AccessToken', {id: '$anonymous'}, optionalHandler);
 }
 
 _describe.whenCalledRemotely = function(verb, url, data, cb) {

